Gemfile.lock

```diff
@@ -1,42 +1,44 @@
 GIT
   remote: git://github.com/voormedia/flipflop.git
-  revision: e8a6724c5fac018e74911c01fc24a6849459ba67
+  revision: 92e3512c6cccd3ce0e3595c861fd912c2e348f94
```

Comment:
One good habit to keep is separating version upgrades from functional changes, except where they are codependent (i.e., doing one without the other will cause things to break) and making a temporary bridge that can work with both versions is outsized compared to syncing up the smallest change to both sides.

It looks like all of these version upgrades are hidden within the config/logging changes. They should have been on a separate commit/pull request. Maybe this came from an incidental `git add .`?